### PR TITLE
Newservice #4

### PR DESCRIPTION
**Pull request by bijanmmarkes**
The purpose of the pull request is to finally have a tagged branch of GTFS-Manager which is completely independent of Transitfeeds and allows the intake of any GTFS endpoint.

- [x] Removed dependency on Transitfeeds
- [x] Standardized config file using JSON
- [x] Updated build to use 6GB of memory rather than 3 to prevent memory issues when compiling

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,9 +3,35 @@ This python script manages GTFS updates by merging the two GTFS feeds. It downlo
 
 ```
 Usage:
```
Comment (jpham93) [inline README.md:5]: Great documentation. Clear instructions and explanation.

**Review by jpham93 — APPROVED** [review]
Very nice. Clean, detailed code with a lot of documentation. Much appreciated!

File: etc/supervisor/conf.d/vta.amigocloud.com.conf [hunk 2]
```diff
@@ -0,0 +1,53 @@
+[program:modeify]
```
Comment (linuxwebexpert) [inline etc/supervisor/conf.d/vta.amigocloud.com.conf]: :x: 
- [ ] Please fix this supervisor file, it has services and events for things that are not related to OTP, leftover crumbs of code that has been deprecated or split separately out for the modeify aka trips.vta.org project.

File: etc/supervisor/conf.d/vta.amigocloud.com.conf [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+command=/home/ubuntu/modeify/runmodeify.sh
+directory=/home/ubuntu/modeify
+autostart=true
+autorestart=true
+stdout_logfile=/var/log/supervisor/vta_modeify.log
+redirect_stderr=true
+
+
+[program:vta_gunicorn]
```
Comment (linuxwebexpert) [inline etc/supervisor/conf.d/vta.amigocloud.com.conf]: :x:
- [ ] No Gnunicorn server running - just a Java Grizzly server, sorry no mythical :unicorn: here, just a big :bear: 

https://javaee.github.io/grizzly/

File: etc/supervisor/conf.d/vta.amigocloud.com.conf [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+autostart=true
+autorestart=true
+stdout_logfile=/var/log/supervisor/vta_otp.log
+redirect_stderr=true
+startsecs=60
+stopwaitsecs=60
+stopsignal=KILL
+stopasgroup=true
+
+[program:vta_otp_monitor]
```
Comment (linuxwebexpert) [inline etc/supervisor/conf.d/vta.amigocloud.com.conf:14]: :question: 
- [ ] Please find out if we are actually using this script, I think it's disabled unless `supervisor` is executing...

File: etc/supervisor/conf.d/vta.amigocloud.com.conf [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+command=/srv/vta.amigocloud.com/monitor_log.sh
+directory=/srv/vta.amigocloud.com
+autostart=true
+autorestart=true
+stdout_logfile=/var/log/supervisor/vta_log_monitor.log
+redirect_stderr=true
+
+[program:vta_websockets]
```
Comment (linuxwebexpert) [inline etc/supervisor/conf.d/vta.amigocloud.com.conf]: :x: 
- [ ] Please remove - no websockets since 2017 / 2018 update to 2.x branch

File: etc/supervisor/conf.d/vta.amigocloud.com.conf [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+directory=/srv/vta.amigocloud.com/vtademo/websockets
+user=www-data
+stdout_logfile=/var/log/supervisor/vta_websockets.log
+redirect_stderr=true
+autostart=true
+autorestart=true
+startsecs=10
+stopwaitsecs=600
+
+[group:vta]
```
Comment (linuxwebexpert) [inline etc/supervisor/conf.d/vta.amigocloud.com.conf:22]: - [ ] Please fix services for `vta` group and update this file - retest - and then please comment with results

File: load_data.sh [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+###
+#
+# GTFS Manager Load Data and Rebuild OpenTripPlanner Graph object daily script
+#
+# @author LinuxWebExpert - see GitHub
+# @date 10/01/2017
+# @see https://github.com/vta/
+#
+###
+MAILTO=""
```
Comment (linuxwebexpert) [inline load_data.sh:11]: :ballot_box_with_check: 
- [ ] Let's make certain `postfix` has been configured on the machine and to edit this file in production
- [ ] In staging and production, we should add a DevOps team email alias, to catch reports sent by this cronjob

See examples here - http://man7.org/linux/man-pages/man5/crontab.5.html

File: load_data.sh [hunk 8]
```diff
@@ -0,0 +1,71 @@
...
+    # Copy the new Graph.obj to OpenTripPlanner default directory
+    cp -v $DATA_DIR/Graph.obj $DATA_DIR/graphs/default/
+    res=$?
+    if test "$res" != "0"; then
+        printf "ERROR Cannot copy new Graph.obj to default directory - exit code: $res"
+    fi
+fi
+
+# Restart the supervisor service to reload the new Graph.obj
+#/usr/bin/supervisorctl restart vta:vta_otp
```
Comment (linuxwebexpert) [inline load_data.sh]: :ballot_box_with_check: 
- [ ] In staging and production we need to enable `supervisor` service to watch things, so this is needed actually

**Review by linuxwebexpert — CHANGES_REQUESTED** [review]
Overall looks very much like the first PR from a long time ago, some improvements, and some extra items that should be removed before merging to master.  Please see my comments, easy things to clean up and test, then we can check again to make certain it's nice, neat and clean.

Also our config.json file I think has more information, like API keys, etc.. and the master should be in our Vault for security.

**Review by linuxwebexpert — APPROVED** [review]
:+1: 
Nice work @jpham93 :100: 

Looks clean, I'm good with the changes. Let's keep in mind we're still working from a `otp.tar.gz` of production files and we need to upgrade to latest [OTP - *see changelog*](http://docs.opentripplanner.org/en/latest/Changelog/) from source or binary, the source is my preference so we learn how to best configure for VTA and the Bay Area, using the configuration parameters.
